requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.8
+# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
Do the requirements need >= Python 3.10 now?

Reply:
No

Reply:
sweet